.github/workflows/tutorial-quickstarts.yml

```diff
@@ -0,0 +1,56 @@
+name: tutorial-quickstarts
+on:
+  schedule:
+    - cron: "0 0/1 * * *"
+  pull_request:
+    branches:
+      - main
+    paths:
+      - tutorials/quickstarts/**
+jobs:
+  build:
+    runs-on: ubuntu-latest 
+    steps:
+    - name: check out repo
+      uses: actions/checkout@v2
+    - name: setup python
+      uses: actions/setup-python@v2
+      with:
+        python-version: "3.6"
```

Comment:
It was the version defined in the Compute Instance "Python 3.6 AzureML" environment, which was why I added it. I'm not sure if it is required, and it could probably be removed.